src/seer/automation/codebase/storage_adapters.py

```
@@ -85,6 +89,12 @@ def save_to_storage(self):
 
         return True
 
+    def delete_from_storage(self):
+        storage_path = self.get_storage_location(self.repo_id, self.namespace_slug)
+        shutil.rmtree(storage_path, ignore_errors=True)
```

I think we should avoid silent failures. ignore_errors should be False? Also, will ignoring errors cause unwanted side effects due to left over files?

---

Maybe we need to add a check for existence to avoid failure if there is nothing to delete?

---

Hmm in this case that the filesystem storage adapter is only used for local dev for now we can just have a check and surface any errors